src/test/kotlin/io/snyk/plugin/ui/toolwindow/SnykToolWindowSnykScanListenerLSTest.kt

```diff
@@ -174,12 +174,12 @@ class SnykToolWindowSnykScanListenerLSTest : BasePlatformTestCase() {
         TestCase.assertEquals(rootTreeNode.children().toList()[1].toString(), " Code Security")
         TestCase.assertEquals(rootTreeNode.children().toList()[2].toString(), " Code Quality")
         TestCase.assertEquals(
-            "✋ 1 vulnerability found by Snyk, 0 ignored",
+            "✋ 1 issue found by Snyk, 0 ignored",
             rootTreeNode.children().toList()[4].toString(),
         )
         TestCase.assertEquals(
             rootTreeNode.children().toList()[5].toString(),
-            "⚡ 1 vulnerabilities can be fixed automatically",
+            "⚡ 1 issues can be fixed automatically",
```

Comment:
Small change needed here. If it's one issue it should read "⚡ 1 issue can be fixed automatically", if more than one then it should read "⚡ 2 issues can be fixed automatically".
i.e. issue for 1, issues for more than 1